index.html

```diff
@@ -0,0 +1,226 @@
+                                    <li class="tech">css</li>
+                                    <li class="tech">JavScript</li>
+                                    <li class="tech">html</li>
+                                </ul>
+                                <a href="" class="btn btn__seeMore">See More</a>
+                            </div>
+
+                        </div>
+                    </div>                
+                </article>
+
+                <!-- Project card 3 -->
+                <article class="project1">
+                    <div class="container">
+                        <div class="card">
+                            <div class="card__img">
+                                <img src="/images/Card/Img Plaholder.svg" alt="Placeholder">
+                            </div>
+                            <div class="project__content">
+                                <h3 class="proj__title">Multi-Post Stories</h3>
+                                <ul class="tech__stack">
+                                    <li class="tech">Ruby on Rails</li>
+                                    <li class="tech">css</li>
+                                    <li class="tech">JavScript</li>
+                                    <li class="tech">html</li>
+                                </ul>
+                                <a href="" class="btn btn__seeMore">See More</a>
+                            </div>
+
+                        </div>
+                    </div>                
+                </article>
+
+                <!-- Project card 4 -->
+                <article class="project1">
+                    <div class="container">
+                        <div class="card">
+                            <div class="card__img">
+                                <img src="/images/Card/Img Plaholder.svg" alt="Placeholder"> 
+                            </div>
+                            <div class="project__content">
+                                <h3 class="proj__title">Multi-Post Stories</h3>
+                                <ul class="tech__stack">
+                                    <li class="tech">Ruby on Rails</li>
+                                    <li class="tech">css</li>
+                                    <li class="tech">JavScript</li>
+                                    <li class="tech">html</li>
+                                </ul>
+                                <a href="" class="btn btn__seeMore">See More</a>
+                            </div>
+
+                        </div>
+                    </div>                
+                </article>
+                <!-- Project card 5 -->
+                <article class="project1">
+                    <div class="container">
+                        <div class="card">
+                            <div class="card__img">
+                                <img src="/images/Card/Img Plaholder.svg" alt="Placeholder">
```

Comment:
[OPTIONAL] 
- [ ] Kindly make sure you use the correct path for your images as you can see they are not displaying correctly.
`hint: you can remove the space between the name you gave your images`